src/pages/releases.md

```diff
@@ -85,31 +85,12 @@ MSSQL requires the use of the [tedious] npm package.
 Sequelize uses the `sqlite3` npm library.  
 [Read more about this here](/docs/v7/other-topics/dialect-specific-things/#sqlite).
 
-:::caution
-
-sqlite3 has not been released since February 2021, and has security vulnerabilities and installation issues on newer platforms.
-
-We recommended using the [@vscode/sqlite3](https://github.com/microsoft/vscode-node-sqlite3) fork 
-by replacing the version with `npm:@vscode/sqlite3@5.0.7` like in the example below.
-
-It is supported in both v6 & v7.
-
-```json
-{
-  "dependencies": {
-    "sqlite3": "npm:@vscode/sqlite3@5.0.7"
-  }
-}
-```
-
-:::
-
 | Sequelize   | [sqlite3]                                                      |
 |-------------|----------------------------------------------------------------|
-| 7 (alpha)   | `npm:@vscode/sqlite3@^5.0.7`                                   |
-| 6 (current) | `npm:@vscode/sqlite3@^4.0.12`, or `npm:@vscode/sqlite3@^5.0.7` |
```

Comment:
`sqlite3@^4.0.12` only exists on the vscode fork, the new sqlite3 release is only applicable to 5.x.x

Reply:
Shall we make it 4.0.9? https://www.npmjs.com/package/sqlite3
Or why did you use 4.0.12?

Reply:
It's the release that fixed the issues with the 4.x branch. We can declare we support `>4.0.0` but that using either `>=5.0.3` or `npm:@vscode/sqlite3@^4.0.12` is recommended?

Reply:
Yes, I'll update the v6 docs and this table to reflect this later today